vpr/src/route/rr_graph.cpp

```diff
@@ -572,13 +591,15 @@ static void build_rr_graph(const t_graph_type graph_type,
         }
     }
 
+    // Not sure what perturbation is. This may also need to be modified since it uses sets_per_seg_type.
```

Comment:
There are a few comments like this that I used for notes that will be removed in a future commit.